test/gnina/test_min.py

```diff
@@ -44,23 +44,45 @@ def are_similar(xyz, sdf):
 #output should be similar to receptor
 assert are_similar('data/C.xyz',outfile)
 
+rmout()
```

Comment:
I would think that a good test would be score_only and making sure the result is in fact cnn+vina (for 1.0) and cnn+X*vina for empirical weight X